ui-react/src/components/topBar/NavButton.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { Button } from 'react-bootstrap';
+import navEllipseIcon from "../../assets/navEllipse.svg"
+
+function NavButton({onClick, text}) {
+  return (
+    <div>
+     <img src={navEllipseIcon} alt='navEllipse' style={{cursor: "pointer"}}/>
+      <Button variant="light" onClick={onClick} style={{backgroundColor:"transparent", borderWidth:0}}>{text}</Button> 
+    </div>
```

Comment:
I believe that the proper solution would be:
```suggestion
      <Button variant="light" onClick={onClick} style={{backgroundColor:"transparent", borderWidth:0,cursor: "pointer", backgroundImage:navEllipseIcon}}>{text}</Button> 
```
This way, the button and the icon are clickable and the whole thing is more semantic and accessible. 

An img tag should not be used unless the image itself is semantically an element which we would like to provide a blind person description for (alt="three dogs playing with each other") in order to make the page contents make sense for someone who can't see it.

 An icon is not something you would want to describe for a blind person, that's why background will be a better option.

Keep in mind i haven't looked at the figma so this may need some size adjustment for the button to make it fit it's own background image and such